pallets/wormhole/src/lib.rs

```diff
@@ -43,114 +64,27 @@ pub mod pallet {
         /// Weight information for pallet operations.
         type WeightInfo: WeightInfo;
 
-        /// Account ID used as the "from" account when creating transfer proofs for minted tokens
-        #[pallet::constant]
-        type MintingAccount: Get<Self::AccountId>;
+        type WeightToFee: WeightToFee<Balance = <Self as BalancesConfig>::Balance>;
+
+        type FeeReceiver: OnUnbalanced<
+            <BalancesPallet<Self> as Currency<Self::AccountId>>::NegativeImbalance,
+        >;
     }
 
     pub trait WeightInfo {
         fn verify_wormhole_proof() -> Weight;
-        fn verify_wormhole_proof_with_used_nullifier() -> Weight;
-        fn verify_wormhole_proof_deserialization_failure() -> Weight;
-        fn verify_wormhole_proof_empty_data() -> Weight;
```

Comment:
This is actually the error states, and Substrate benchmarking framework is designed to measure the cost only of successful extrinsic execution. When an extrinsic returns an error (like `NullifierAlreadyUsed` or `ProofDeserializationFailed`), which it does in the above cases, it treats this as a benchmark failure rather than a success